server/app.js

```diff
@@ -243,6 +280,7 @@ app.get('/css/:id', asyncHandler(getBrew('share')), (req, res)=>{getCSS(req, res
 //User Page
 app.get('/user/:username', async (req, res, next)=>{
 	const ownAccount = req.account && (req.account.username == req.params.username);
+	console.log(req.account);
```

Comment:
Just a note we should eventually remove all these logs of user account details after debugging this feature is complete.

Reply:
This one can be removed now, the rest yes of course